src/applications/vaos/services/appointment/transformers.js

```diff
@@ -25,10 +25,13 @@ function getAppointmentType(appt) {
     return APPOINTMENT_TYPES.ccRequest;
   } else if (appt.typeOfCareId) {
     return APPOINTMENT_TYPES.request;
-  } else if (appt.clinicId || appt.vvsAppointments) {
-    return APPOINTMENT_TYPES.vaAppointment;
-  } else if (appt.appointmentTime) {
+  } else if (appt.distanceEligibleConfirmed || appt.communityCare === true) {
```

Comment:
Do we need to check `distanceEligibleConfirmed` as well as the `communityCare` property?

Reply:
I switched back to `appointmentTime` instead of `distanceEligibleConfirmed` for the conditional check